### DISCUSSION TITLE
TextEditor: blur and change events

### DISCUSSION THREAD

**Discussion by bjornerikolsen** [primefaces/community#2563] [PrimeFaces] [answered]
After upgrading to Primefaces 13 I noticed that for TextEditor change event is not fired for every keystroke like it was before in Primefaces 12. Now a change event is fired after blur/focus changed to another component. Is this as expected? Changes made: https://github.com/primefaces/primefaces/issues/10112

**Comment by melloware** [comment, marked as answer]
Yes this is the new expected behavior in PF13+

**Reply by bjornerikolsen** [reply]
Thanks for the reply. Is there another way around this to get the behaviour from PF12 other than override the PrimeFaces.widget.DeferredWidget.extend function?

**Reply by melloware** [reply]
I don't think so other than as you suggested MonkeyPatching the component to add the functionality the way you want it.
